src/main/kotlin/com/github/quiltservertools/ledger/config/NetworkingSpec.kt

```diff
@@ -4,7 +4,4 @@ import com.uchuhimo.konf.ConfigSpec
 
 object NetworkingSpec : ConfigSpec() {
     val networking by required<Boolean>()
-    val allowByDefault by required<Boolean>()
```

Comment:
Confirm this doesn't cause errors with old configs. Also I don't think you removed them from the actual default config